### PR TITLE
sulu:document:initialize command

### PR DESCRIPTION
**Pull request by dantleech**
Modified documentation to remove references to `sulu:webspaces:init` and `sulu:phpcr:init`.

### CODE REVIEW

File: cookbook/create-new-webspace.rst [hunk 1]
```diff
@@ -11,11 +11,14 @@ similar to the `sulu_io.xml.dist` file in this folder.
     extension.
 
 To activate the webspace within sulu with a `prod` or `stage` environment
-you have to clear the cache with the command:
-`app/console clear:cache -e <environment>`
+you have to clear the cache with the command::
```
Comment (danrot) [inline cookbook/create-new-webspace.rst]: One colon would be enough :wink:

File: cookbook/create-new-webspace.rst [hunk 2]
```diff
@@ -11,11 +11,14 @@ similar to the `sulu_io.xml.dist` file in this folder.
...
 
-To initiate the database for the new webspace run the command
-`app/console sulu:phpcr:init && app/console sulu:webspaces:init`.
+    $ app/console clear:cache -e <environment>
```
Comment (leehaskings) [inline cookbook/create-new-webspace.rst]: Should also read "cache:clear" not "clear:cache" :)

Comment (dantleech) [inline cookbook/create-new-webspace.rst]: Indeed :)

**Comment by dantleech** [comment]
Updated.

**Pull request merged**